packages/airnode-deployer/config/config.json.example

```diff
@@ -1,6 +1,7 @@
 {
   "chains": [
     {
+      "maxConcurrentTransactions": 100,
```

Comment:
Talked with Burak on call and we agreed that this field should be called `maxConcurrency` instead.